docs/en/observability/monitor-infra/host-details-partial.asciidoc

```diff
@@ -0,0 +1,149 @@
+
+* Host information
+* Cloud information
+* Agent information
+
+All of this information can help when investigating events—for example, filtering by operating system or architecture.
+====
+
+[%collapsible]
+.*Processes*
+====
+
+[role="screenshot"]
+image::images/processes-overlay.png[Host processes]
+
+The *Processes* tab lists the total number of processes (`system.process.summary.total`) running on the host,
+along with the total number of processes in these various states:
+
+* Running (`system.process.summary.running`)
+* Sleeping (`system.process.summary.sleeping`)
+* Stopped (`system.process.summary.stopped`)
+* Idle (`system.process.summary.idle`)
+* Dead (`system.process.summary.dead`)
+* Zombie (`system.process.summary.zombie`)
+* Unknown (`system.process.summary.unknown`)
+
+The processes listed in the *Top processes* table are based on an aggregation of the top CPU and the top memory consuming processes.
+The number of top processes is controlled by `process.include_top_n.by_cpu` and `process.include_top_n.by_memory`.
+
+|=== 
+
+| *Command* | Full command line that started the process, including the absolute path to the executable, and all the arguments (`system.process.cmdline`).
+| *PID* | Process id (`process.pid`).
+| *User* | User name (`user.name`).
+| *CPU* | The percentage of CPU time spent by the process since the last event (`system.process.cpu.total.pct`).
+| *Time* | The time the process started (`system.process.cpu.start_time`). 
+| *Memory* | The percentage of memory (`system.process.memory.rss.pct`) the process occupied in main memory (RAM). 
+| *State* | The current state of the process and the total number of processes (`system.process.state`). Expected values are: `running`, `sleeping`, `dead`, `stopped`,
+`idle`, `zombie`, and `unknown`.
+
+|=== 
+====
+
+[%collapsible]
+.*Logs*
+====
+
+[role="screenshot"]
+image::images/logs-overlay.png[Host logs]
+
+The *Logs* tab displays logs relating to the host that you have selected. By default, the logs tab displays the following columns. 
+
+|=== 
+
+| *Timestamp* | The timestamp of the log entry from the `timestamp` field. 
+
+| *Message* | The message extracted from the document.
+The content of this field depends on the type of log message.
+If no special log message type is detected, the {ecs-ref}/ecs-base.html[Elastic Common Schema (ECS)]
+base field, `message`, is used.
+
+|=== 
+
+You can customize the logs view by adding a column for an arbitrary field you would like
+to filter by. For more information, refer to <<customize-stream-page,Customize Stream>>.
+To view the logs in the {logs-app} for a detailed analysis, click *Open in Logs*.
+====
+
+[%collapsible]
+.*Anomalies*
+====
+
+[role="screenshot"]
+image::images/anomalies-overlay.png[Anomalies]
+
+The *Anomalies* table displays a list of each single metric {anomaly-detect} job for the specific host. By default, anomaly
+jobs are sorted by time to show the most recent job. 
```

Comment:
```suggestion
jobs are sorted by time, showing the most recent jobs first. 
```
Maybe not necessary, but I got hung up on this sentence, thinking of "time to show the most recent job" was a way to sort.